src/util/testUtil.ts

```diff
@@ -60,3 +61,60 @@ export async function signWithAlice(signingPayload: string): Promise<string> {
 
   return signature;
 }
+
+/**
+ * @ignore
+ */
+export const TEST_BOND_TX_INFO: TxInfoBond = {
```

Comment:
Is this test case from a real example of a bond extrinsic?

Reply:
Like one pulled from the chain? No.

Reply:
Most of the data I copied from the balance transfer example, so you'd have to ask Amaury. I just changed controller, value, and payee, which are not from a real example.

Reply:
Actually, I don't think we can't use real world extrinsics, because the tests use Alice's keys to sign transactions. I'd do:
- run test once, remember the signatures
- copy/paste and hardcode them into tests, make tests pass

The tests will then act like regression tests for future commits.